site/content/en/docs/tasks/run/multikueue/job.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Run Kubernetes Job in Multi-Cluster"
+linkTitle: "Kubernetes Job"
+weight: 2
+date: 2024-11-05
+description: >
+  Run a MultiKueue scheduled Kubernetes Job.
+---
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+For the ease of setup and use we recommend using at least Kueue v0.8.1 and Kubernetes v1.30.0.
+
+## Run Kubernetes Job example
+
+Once the setup is complete you can test it by running the [`job-sample.yaml`](/docs/tasks/run/jobs/#1-define-the-job) Job. 
```

Comment:
Could you embed the YAML file similar to https://github.com/kubernetes-sigs/kueue/blob/337e8712fbb0c582819effe780f70ad0429b7ceb/site/content/en/docs/concepts/topology_aware_scheduling.md?plain=1#L74?